scheme/arm-cca/realm.go

```diff
@@ -0,0 +1,119 @@
+// Copyright 2024 Contributors to the Veraison project.
+// SPDX-License-Identifier: Apache-2.0
+
+package arm_cca
+
+import (
+	"bytes"
+	"encoding/base64"
+	"fmt"
+
+	"github.com/veraison/ccatoken"
+	"github.com/veraison/ear"
+	"github.com/veraison/services/handler"
+	"github.com/veraison/services/log"
+	"github.com/veraison/services/scheme/common/arm"
+	realm "github.com/veraison/services/scheme/common/cca/realm"
+)
+
+type CCARealmAttester struct {
+}
+
+func (CCARealmAttester) PerformAppraisal(
```

Comment:
ditto here: https://github.com/veraison/services/pull/233/files#r1644911152

Reply:
Changed in https://github.com/veraison/services/commit/974a19615feee71e7c2378a8075d9617efe11513